src/3d/symbols/qgsline3dsymbol_p.cpp

```diff
@@ -150,4 +154,105 @@ Qt3DRender::QGeometryRenderer *QgsLine3DSymbolEntityNode::renderer( const Qgs3DM
   return renderer;
 }
 
+
+Qt3DRender::QGeometryRenderer *QgsLine3DSymbolEntityNode::rendererSimple( const Qgs3DMapSettings &map, const QgsLine3DSymbol &symbol, const QgsVectorLayer *layer, const QgsFeatureRequest &request )
+{
+  QVector<QVector3D> vertices;
+  vertices << QVector3D();  // the first index is invalid, we use it for primitive restart
+  QVector<unsigned int> indexes;
+
+  QgsPoint centroid;
+  QgsPointXY origin( map.origin().x(), map.origin().y() );
+  QgsFeature f;
+  QgsFeatureIterator fi = layer->getFeatures( request );
+  while ( fi.nextFeature( f ) )
+  {
+    if ( f.geometry().isNull() )
+      continue;
+
+    if ( symbol.altitudeBinding() == AltBindCentroid )
+      centroid = QgsPoint( f.geometry().centroid().asPoint() );
+
+    QgsGeometry geom = f.geometry();
+    const QgsAbstractGeometry *g = geom.constGet();
+    if ( QgsLineString *ls = qgsgeometry_cast<QgsLineString *>( g ) )
+    {
+      for ( int i = 0; i < ls->vertexCount(); ++i )
+      {
+        QgsPoint p = ls->vertexAt( QgsVertexId( 0, 0, i ) );
+        float z = Qgs3DUtils::clampAltitude( p, symbol.altitudeClamping(), symbol.altitudeBinding(), symbol.height(), centroid, map );
+        vertices << QVector3D( p.x() - map.origin().x(), z, p.y() - map.origin().y() );
```

Comment:
On my machine, this leads to lines wrongly placed. The lines are properly placed when I changed y bit to `map.origin().y() - p.y()`.

![fixed](https://user-images.githubusercontent.com/1728657/43375562-0c0017ce-93e0-11e8-83b4-d8fe201fe0b0.png)